wiki/Difficulties/en.md

```diff
@@ -11,13 +11,14 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmaps).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete the beatmap. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult the beatmap is.
```

Comment:
This reads worse to me after the change due to the stylistic repetition:

> a *beatmap's* difficulty (...) to complete the *beatmap*

> star rating of a *beatmap* (...) how difficult the *beatmap* is

The previous version using pronouns avoided this and was unambiguous in my opinion.